src/analyzer/git-caller.server.ts

```diff
@@ -14,39 +15,16 @@ export enum ANALYZER_CACHE_MISS_REASONS {
   DATA_VERSION_MISMATCH = "Outdated cache"
 }
 
-export type RawGitObjectType = "blob" | "tree" | "commit" | "tag"
-export type RawGitObject = {
-  hash: string
-  type: RawGitObjectType
-  idk: string
-  value: string
-}
-
 export class GitCaller {
   private useCache = true
-  private repo: string
-  public branch?: string
   private catFileCache: Map<string, string> = new Map()
-  private blameCache: Map<string, string> = new Map()
-
-  private static instance: GitCaller | null = null
-
-  static initInstance(repo: string) {
-    if (!GitCaller.instance || GitCaller.instance.repo !== repo) {
-      GitCaller.instance = new GitCaller(repo)
-    }
-  }
-
-  static destroyInstance() {
-    GitCaller.instance = null
-  }
 
-  static getInstance(): GitCaller {
-    if (!GitCaller.instance) {
-      throw Error("ObjectDeflator not initialized")
-    }
-    return GitCaller.instance
-  }
+  // eslint-disable-next-line no-useless-constructor
```

Comment:
Can this constructor be omitted? Not sure.

Reply:
Dunno how that would be done